src/ocean/net/server/connection/IConnectionHandler.d

```diff
@@ -321,6 +321,14 @@ abstract class IConnectionHandler : IConnectionHandlerInfo,
                 this.error(this.socket_error.setSock("error closing connection", __FILE__, __LINE__));
             }
 
+            // The socket is closed below. However, this does not always mean
+            // that it is unregistered from epoll -- in situations where the
+            // process has forked, the fork's reference to the fd will prevent
```

Comment:
I suggest: "the fork's reference to the underlying kernel's file description" to emphasize that there are two file descriptors pointing to the same underlying file description.

Reply:
Updated.